### PR TITLE
Upload 746/post hook to ps

### PR DESCRIPTION
**Pull request by rohitpanwar**

### CODE REVIEW

File: tus/file-hooks/upload-status/post_receive_bin.py [hunk 1]
```diff
@@ -135,8 +120,8 @@ def main(argv):
             size = arg
         elif opt in ("-m", "--metadata"):
             metadata = arg
-    try:
-        post_receive(tus_id, int(offset), int(size), metadata)
+    try:        
+        asyncio.run(post_receive(tus_id, int(offset), int(size), metadata))
```
Comment (cfarmer-fearless) [inline tus/file-hooks/upload-status/post_receive_bin.py:124]: What does the `asyncio.run` do?

File: tus/file-hooks/upload-status/post_receive_bin.py [hunk 2]
```diff
@@ -111,8 +82,22 @@ def post_receive(tguid, offset, size, metadata_json):
         # convert metadata json string to a dictionary
         metadata_json_dict = ast.literal_eval(metadata_json)
```
Comment (cfarmer-fearless) [inline tus/file-hooks/upload-status/post_receive_bin.py:83]: Why are we deserializing this json string just to re-serialize it on line 99 with the `json.dumps` function?

File: tus/file-hooks/upload-status/post_receive_bin.py (hunk 1 above)
Comment (cfarmer-fearless) [reply tus/file-hooks/upload-status/post_receive_bin.py:124]: Future reference: async/await syntax for Python.  Shouldn't cause performance issues.

File: tus/file-hooks/upload-status/post_receive_bin.py (hunk 2 above)
Comment (rohitpanwar) [reply tus/file-hooks/upload-status/post_receive_bin.py]: made the change to add up json strings instead

File: tus/file-hooks/upload-status/post_receive_bin.py (hunk 1 above)
Comment (rohitpanwar) [reply tus/file-hooks/upload-status/post_receive_bin.py:111]: leaving this as is as we decided

**Review by cfarmer-fearless — APPROVED** [review]

**Pull request merged**